app/src/main/java/com/github/multimatum_team/multimatum/adaptater/DeadlineAdapter.kt

```diff
@@ -34,31 +39,37 @@ Class which is used to show the deadline in a clear list.
 Based on the tutorial:
 https://www.raywenderlich.com/155-android-listview-tutorial-with-kotlin
  */
+enum class FilterState{
+    /**
+     * state to filter deadlines in groups
+     */
+    GROUPS,
+
+    /**
+     * state to filter deadlines not in groups
+     */
+    MINE,
+
+    /**
+     * don't filter deadlines
+     */
+
+    ALL
+}
```

Comment:
It is a detail, but for readability purposes, I suggest to put these comments as "line comments " and reduce the space between elements.